### ISSUE TITLE
E251 false positive with some typings

### ISSUE THREAD

**Issue by arthurwhite** [PyCQA/pycodestyle#806]
Same as old #496.

I've read that #508 should have fixed this kind of problem but it's not the case for special forms of typings like `Union`:

```python
from typing import Union


def foobar(foo: Union[int, str] = 1):
    pass
```

Returns `E251:unexpected spaces around keyword / parameter equals`.
And `# noqa` won't disable this warning.

Using `pep8` version 1.7.1.

**Comment by sigmavirus24** [comment]
`pep8` 1.7.1 is no longer updated. Please switch to `pycodestyle`

**Comment by arthurwhite** [comment]
OK, I was misled by the VS Code Python extension which still requires `pep8` by default.

I installed `pydocstyle` and added `"python.linting.pep8Path": "pycodestyle"` to my settings.
Now it works. 👍

Thanks!
